R/utils.R

```diff
@@ -49,7 +49,7 @@ getRefRun <- function(peptideScores, applyFun=lapply){
 #'
 #' License: (c) Author (2020) + GPL-3
 #' Date: 2020-04-08
-#' @importFrom data.table rbindlist set setkeyv
+#' @importFrom data.table rbindlist set setkeyv setcolorder
 #' @importFrom bit64 NA_integer64_
 #' @inheritParams alignTargetedRuns
 #' @param precursors (data-frames) Contains precursors and associated transition IDs.
```

Comment:
Add runType definition here. Warning from `devtools::check()`

Reply:
Done. I will make the example dataset smaller.